ts/components/wallet/PaymentSummaryComponent.tsx

```diff
@@ -32,13 +42,45 @@ const styles = StyleSheet.create({
     flexDirection: "column",
     flex: 1,
     paddingRight: 4
+  },
+  flex: {
+    flex: 1
+  },
+  paymentOutcome: {
+    flexDirection: "row"
+  },
+  text11: {
+    paddingLeft: 8
   }
 });
 
 /**
  * This component displays the transaction details
  */
 export const PaymentSummaryComponent = (props: Props) => {
+  const renderItem = (label: string, value: string) => (
+    <React.Fragment>
+      <Text small={true} style={props.dark && styles.lighterGray}>
+        {label}
+      </Text>
+      <Text bold={true} dark={!props.dark} white={props.dark}>
+        {value}
+      </Text>
+      <View spacer={true} />
+    </React.Fragment>
+  );
+
+  const tt = props.paymentStatus && (
```

Comment:
```suggestion
  const paymentStatus = props.paymentStatus && (
```

Reply:
Updated!

Here the screen after https://github.com/pagopa/io-app/pull/1800/commits/0300aa988de205cec275d1f1e29549f6003aca1f is commited:
<img width="356" alt="image" src="https://user-images.githubusercontent.com/38431762/82917737-15dc4000-9f74-11ea-8f6b-0b3fb309fedd.png">